packages/airnode-admin/src/implementation.ts

```diff
@@ -1,13 +1,50 @@
 import * as airnodeAbi from '@api3/airnode-abi';
 import { AirnodeRrp, RequesterAuthorizerWithAirnode } from '@api3/airnode-protocol';
 import { ethers } from 'ethers';
+import { Arguments } from 'yargs';
 
 const assertAllParamsAreReturned = (params: object, ethersParams: any[]) => {
   if (Object.keys(params).length !== ethersParams.length) {
     throw new Error(`SDK doesn't return some of the parameters!`);
   }
 };
 
+/**
+ * Parses Ethers transaction override options from CLI arguments.
+ * @param args The yargs inferred CLI arguments object
+ * @returns The parsed overrides object with values compatible with ethers
+ */
+export const parseTransactionOverrides = (
+  args: Arguments | { [key: string]: string }
+): ethers.Overrides | ethers.PayableOverrides => {
+  if ((args['max-fee'] || args['max-priority-fee']) && args['gas-price'])
+    throw new Error('EIP-1559 transactions cannot have gas-price argument');
+  if ((args['max-fee'] && !args['max-priority-fee']) || (!args['max-fee'] && args['max-priority-fee']))
+    throw new Error('EIP-1559 transactions require both max-fee and max-priority-fee arguments');
+  if (args['value'] && ethers.utils.parseEther(args['value'] as string).lt(0))
```

Comment:
Do we need to check this? We are not checking this for the other ones (and they need to be non zero as well).
I think it's fine to let ethers check this.

Reply:
We could probably drop all of these checks if it's fine to rely on ethers checking them?

Reply:
Do they also check that you can't mix the eip1559 and legacy params?

Reply:
Yes it also checks that :+1: